server.js

```diff
@@ -133,8 +133,9 @@ function handler(from, to, message) {
 
   // watch for github issue links to any repository
   var issues_re = /https:\/\/github\.com\/([\w\-]+)\/([\w\-]+)\/(issues|pull)\/(\d+)/g;
+  var reviewable_re = /https:\/\/reviewable\.io\/reviews\/([\w\-]+)\/([\w\-]+)\/(\d+)/g;
   var issues;
-  while ((issues = issues_re.exec(message)) !== null) {
+  while ((issues = (issues_re.exec(message) || reviewable_re.exec(message) )) !== null) {
     var type = issues[3];
     searchGithub("/" + issues[4], issues[1], issues[2], function(error, issue) {
```

Comment:
The indexes are off for the reviewable case, unfortunately. Github links provide [org]/[repo]/[issue|pull]/NNNN, whereas reviewable gives us [org]/[repo]/NNNN.